src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -55,4 +92,4 @@ export default BaseAuthenticatorView.extend({
       }
     });
   }
-});
+});
```

Comment:
nit: newline

Reply:
Fixed in https://github.com/okta/okta-signin-widget/pull/1281